src/main/java/frc/trigon/robot/commands/Commands.java

```diff
@@ -70,4 +83,109 @@ public static Command duplicate(Command command) {
                 command.getRequirements().toArray(Subsystem[]::new)
         );
     }
+
+    /**
+     * @param isDelivery if the robot is shooting the note for a delivery or for a shot
+     * @return a command that sets the robot to shoot at the given target
+     */
+    public static Command getShootAtShootingTargetCommand(boolean isDelivery) {
+        return new ParallelCommandGroup(
+                getPrepareShooterCommand(isDelivery),
+                getFeedNoteForShooterCommand()
+        );
+    }
+
+    public static Command getPrepareShooterCommand(boolean isDelivery) {
+        return new ParallelCommandGroup(
+                getUpdateShootingCalculationsCommand(isDelivery),
+                PitcherCommands.getReachTargetPitchFromShootingCalculationsCommand(),
+                ShooterCommands.getReachTargetShootingVelocityFromShootingCalculationsCommand(),
+                SwerveCommands.getClosedLoopFieldRelativeDriveCommand(
+                        () -> CommandConstants.calculateDriveStickAxisValue(OperatorConstants.DRIVER_CONTROLLER.getLeftY()),
+                        () -> CommandConstants.calculateDriveStickAxisValue(OperatorConstants.DRIVER_CONTROLLER.getLeftX()),
+                        () -> SHOOTING_CALCULATIONS.getTargetShootingState().targetRobotAngle()
+                )
+        );
+    }
+
+    public static Command getWarmForSpeakerShotCommand() {
+        return new ParallelCommandGroup(
+                getUpdateShootingCalculationsCommand(false),
+                PitcherCommands.getReachTargetPitchFromShootingCalculationsCommand(),
+                ShooterCommands.getReachTargetShootingVelocityFromShootingCalculationsCommand()
+        );
+    }
+
+    public static Command getShootCloseShotCommand() {
+        return new ParallelCommandGroup(
+                getPrepareShooterForCloseShotCommand(),
+                getFeedNoteForCloseShotCommand()
+        );
+    }
+
+    public static Command getPrepareShooterForCloseShotCommand() {
```

Comment:
Again, no need for shooter (btw, this is also pitcher), and add the word speaker.